src/FsAutoComplete.Core/Commands.fs

```diff
@@ -1265,111 +1265,165 @@ type Commands() =
-            Some false
-
-          | SyntaxNode.SynTypeDefn(SynTypeDefn(typeInfo = SynComponentInfo(longId = longId; xmlDoc = xmlDoc))) when
-            longIdentContainsPos longId pos && xmlDoc.IsEmpty
-            ->
-            Some false
-
-          | SyntaxNode.SynTypeDefn(SynTypeDefn(
-              typeRepr = SynTypeDefnRepr.Simple(SynTypeDefnSimpleRepr.Record(recordFields = fields), _))) ->
-            let isInLine c =
-              match c with
-              | SynField(xmlDoc = xmlDoc; idOpt = Some ident) when rangeContainsPos ident.idRange pos && xmlDoc.IsEmpty ->
-                Some false
-              | _ -> None
-
-            fields |> List.tryPick isInLine
-
-          | SyntaxNode.SynTypeDefn(SynTypeDefn(
-              typeRepr = SynTypeDefnRepr.Simple(SynTypeDefnSimpleRepr.Union(unionCases = cases), _))) ->
-            let isInLine c =
-              match c with
-              | SynUnionCase(xmlDoc = xmlDoc; ident = SynIdent(ident = ident)) when
-                rangeContainsPos ident.idRange pos && xmlDoc.IsEmpty
-                ->
-                Some false
-              | _ -> None
-
-            cases |> List.tryPick isInLine
-
-          | SyntaxNode.SynTypeDefn(SynTypeDefn(
-              typeRepr = SynTypeDefnRepr.Simple(SynTypeDefnSimpleRepr.Enum(cases = cases), _))) ->
-            let isInLine c =
-              match c with
-              | SynEnumCase(xmlDoc = xmlDoc; ident = SynIdent(ident = ident)) when
-                rangeContainsPos ident.idRange pos && xmlDoc.IsEmpty
-                ->
-                Some false
-              | _ -> None
-
-            cases |> List.tryPick isInLine
-
-          | SyntaxNode.SynModuleOrNamespace(SynModuleOrNamespace(longId = longId; xmlDoc = xmlDoc)) when
-            longIdentContainsPos longId pos && xmlDoc.IsEmpty
-            ->
-            Some false
-
-          | SyntaxNode.SynModule(SynModuleDecl.NestedModule(
-              moduleInfo = SynComponentInfo(longId = longId; xmlDoc = xmlDoc))) when
-            longIdentContainsPos longId pos && xmlDoc.IsEmpty
-            ->
-            Some false
-
-          | SyntaxNode.SynMemberDefn(SynMemberDefn.AutoProperty(ident = ident; xmlDoc = xmlDoc)) when
-            rangeContainsPos ident.idRange pos && xmlDoc.IsEmpty
-            ->
-            Some true
-
-          | SyntaxNode.SynMemberDefn(SynMemberDefn.GetSetMember(
-              memberDefnForGet = Some(SynBinding(xmlDoc = xmlDoc; headPat = SynPat.LongIdent(longDotId = longDotId)))))
-          | SyntaxNode.SynMemberDefn(SynMemberDefn.GetSetMember(
-            memberDefnForSet = Some(SynBinding(xmlDoc = xmlDoc; headPat = SynPat.LongIdent(longDotId = longDotId))))) when
-            rangeContainsPos longDotId.Range pos && xmlDoc.IsEmpty
-            ->
-            Some false
-
-          | _ -> None)
+        match isLowerAstElemWithEmptyPreXmlDoc input pos with
```

Comment:
@dawedawe can you check this merge to make sure I did it correctly?

Reply:
Yep, works as expected 👍